editor/src/core/workers/parser-printer/__snapshots__/parser-printer.spec.ts.snap

```diff
@@ -909,17 +907,16 @@ Object {
             "n",
           ],
           "elementsWithin": Object {},
-          "javascript": "n > 0",
+          "javascript": "n > 0;",
```

Comment:
Why are all these extra line ending semicolons necessary?

Reply:
This is because the javascript has now gone through a babel transpile (whereas before it was just the same as the `originalJavascript`). The `javascript` field should be renamed to `javascriptWithUIDs`, as the naming here is terrible, but I'll do that in a follow up PR